README.md

```diff
@@ -53,9 +53,18 @@ php artisan budget:update
 
 ## Docker
 
-You can run Budget using Docker. Spinning it up (`docker-compose up`) will create 3 containers–for MySQL, PHP and NGINX.
-
-*Disclaimer–currently the Docker setup is missing cronjobs and the queue worker.*
+You can run Budget using Docker Compose. Docker starts 2 containers - one for the database and the other for the application itself.
+
+1. Clone the repository.
+2. Copy the `.env.example` to `.env`, make sure to add an `APP_KEY` and review/replace any other value as you wish.\
+   The `APP_KEY` can be any random value but should be rotated carefully as session cookies are encrypted with this value.
+3. Make sure you have the most up-to-date images.\
+`docker-compose pull`
```

Comment:
For simplification I would just put the lines next to each other, like below. But that's up to you 🙂 

```diff
- 3. Make sure you have the most up-to-date images.\
- `docker-compose pull`
+ 3. Make sure you have the most up-to-date images (`docker-compose pull`)
```